src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -166,7 +166,8 @@ private Optional<ParserResult> importFile(Path file, String importFormat) {
             ImportFormatReader importFormatReader = new ImportFormatReader(
                     preferencesService.getImporterPreferences(),
                     preferencesService.getImportFormatPreferences(),
-                    fileUpdateMonitor);
+                    fileUpdateMonitor,
+                    preferencesService.getCitationKeyPatternPreferences());
```

Comment:
Maybe move it before `fileUpdateMonitor` group preferences together...

IntelliJ supports refactoring for this, so you don't need to do it by hand.